src/resources/shipment.js

```diff
@@ -110,6 +110,14 @@ export default api => (
       return this.rpc('rates', undefined, undefined, 'get');
     }
 
+    async getSmartrates() {
+      this.verifyParameters({
+        this: ['id'],
+      });
+
+      return this.rpc('smartrate', undefined, undefined, 'get');
```

Comment:
We'll probably want to do something like this to future-proof our API:

```suggestion
      const response = this.rpc('smartrate', undefined, undefined, 'get');
      const smartRates = (typeof response.result === 'Array' ? response.result : response) || [];
      return smartRates;
```

Reply:
Do we want to do a protected get?